keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt

```diff
@@ -77,7 +77,7 @@ class ArtifactController(
     @PathVariable name: String,
     @PathVariable type: ArtifactType
   ): List<String> =
-    artifactRepository.versions(DeliveryArtifact(name, type))
+    artifactRepository.versions(name, type)
```

Comment:
We need to be able to query by artifact name and type, because it's not always valid to construct the artifact. How do we know how it's sorted? In the repository, we query for the full artifact to get this information.